tests/psoc6/run_psoc6_tests.sh

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/sh
+
+#
+# psoc6/run_psoc6_tests.sh -ci
+#
+
+
+echo "executing $0 $* ..."
```

Comment:
:) So if we would like to add also the complete test evaluation to the ci_run_test() https://github.com/jaenrig-ifx/micropython/blob/ports/psoc6/tools/ci.sh#L273-L275... we have to previously load the files required by some tests? 
What other obstacle for automation are there?

Reply:
Our flash is mounted at /flash whereas all file operations assume to be at a directory level directly above the tests. So what needs to be done ?
1. create all directories for the tests and their data files on the board, say at /flash/...
2. copy all tests and data files to their respective directories on the board
3. cd to the directory 1 level containing the tests, say /flash
4. execute run-tests.py including these tests